.github/workflows/ci_manager.yml

```diff
@@ -31,10 +31,24 @@ jobs:
               id: check_rs_comments
               run: |
                 git fetch origin ${{ github.base_ref }}
-                if git diff --name-only origin/${{ github.base_ref }} HEAD | grep -qvE '(\.md$|\.css$|\.html$|^AUTHORS$|\.rs)'; then
+                if git diff --name-only origin/${{ github.base_ref }} HEAD | grep -qvE '(\.md$|\.css$|\.html$|^AUTHORS$|\.rs|\.roc)'; then
                     echo "should_run_tests=y" >> $GITHUB_OUTPUT
                 else
-                  if git diff --unified=0 origin/${{ github.base_ref }} HEAD '*.rs' | grep -E --color=never '^[+-]' | grep -qvE '^(\+\+\+|\-\-\-|[+-]\s*($|\/\/|[+-]|\/\*.*\*\/\s*$))'; then
```

Comment:
I've removed a redundant (the second one) instance of `[+-]` from `grep -qvE '^(\+\+\+|\-\-\-|[+-]\s*($|\/\/|[+-]|\/\*.*\*\/\s*$))'`.

This has been confirmed via unit-tests to not have made any difference. Essentially, it was checking for `+` or `-`, which were following a line starting in `+++`, `---`, `+` or `-`.